src/tiled/mainwindow.cpp

```diff
@@ -416,6 +419,8 @@ MainWindow::MainWindow(QWidget *parent, Qt::WindowFlags flags)
     connect(mDocumentManager, SIGNAL(reloadError(QString)),
             this, SLOT(reloadError(QString)));
 
+    connect(mResetDefaultLayout,SIGNAL(triggered(bool)),this,SLOT(resetDefaultLayout()));
```

Comment:
Please prefer modern connect syntax for new connections (eventually I'm going to do a commit changing all of the old ones I guess):
```c++
connect(mResetDefaultLayout, &QAction:triggered, this, &MainWindow::resetDefaultLayout);
```
The new syntax works more reliably when things are renamed / refactored, since the compiler can check the existence of the signals/slots and the compatibility of their arguments.